meshroom/ui/qml/GraphEditor/AttributeItemDelegate.qml

```diff
@@ -222,6 +223,38 @@ RowLayout {
                         else if (drop.hasText && drop.text != '')
                             setTextFieldAttribute(drop.text)
                     }
+
+                }
+                MouseArea {
+                    anchors.fill: parent
+                    acceptedButtons: Qt.AllButtons
+                    onClicked: {
+                        if (mouse.button == Qt.RightButton) {
+                            var menu = menuCopy.createObject(parameterLabel)
+                            menu.parent = parameterLabel
+                            menu.popup()
+                        } else {
+                            textField.forceActiveFocus()
+                        }
+                    }
+
+                    property Component menuCopy : Menu {
+                        MenuItem {
+                            text: "Copy"
+                            enabled: attribute.value != ""
+                            onTriggered: {
+                                Clipboard.clear()
+                                Clipboard.setText(attribute.value)
+                            }
+                        }
+                        MenuItem {
+                            text: "Paste"
+                            enabled: Clipboard.getText() != ""
```

Comment:
Tiny remark here: the "paste" menu should not only appear if the clipboard is not empty, it should also appear if the text field we're right-clicking on is not read-only.

With the current behaviour, we can right-click on an output attribute and have the "paste" menu appearing, but when pasting there, nothing happens (which is the expected behaviour, since output File attributes are read-only). The "paste" menu should be disabled in that case.